python/doc/examples/numerical_methods/optimization/plot_ego.py

```diff
@@ -287,10 +288,11 @@
 result.getIterationNumber()
 
 # %%
-result.getOptimalPoint()
+# we have to use the multi-objective accessors because the objective is 2-d
+result.getFinalPoints()
 
 # %%
-result.getOptimalValue()
+result.getFinalValues()
```

Comment:
I don't understand this change. I compiled your branch and the original commands worked just fine and produced the correct result.